### PR TITLE
Remove Contact Form

### LINKED ISSUE #221: Remove the contact form completely
**Is your feature request related to a problem? Please describe.**
Currently, the contact form is not looking good

**Describe the solution you'd like**
Remove the contact form completely

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by PiyushXCoder**
## Purpose
Closes  #221

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -190,25 +190,7 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
 
     <!-- Contact Section -->
     <div id="contact">
-      <h1>CONTACT ME</h1>
-      <h3>CONTACT <span class="text-info">ME</span></h3>
-
       <div class="formContainer bg-dark p-5 rounded-3">
```
Comment (YoshithaRathnayake) [inline dark/index.html:193]: Can I know why did you keep these lines? @PiyushXCoder

File: dark/assets/css/styles.css [hunk 2]
```diff
@@ -1056,39 +1058,6 @@ html {
...
-  position: absolute;
-  display: block;
-  width: 50px;
-  height: 4px;
-  background: #0563bb;
-  bottom: -26px;
-  left: calc(50% - 20px);
-}
-
 .formContainer {
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1061]: these also from line 1061 - 1094

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 1 above)
Comment (PiyushXCoder) [reply dark/index.html:196]: I thought that also needed to be removed. I have reverted.

Comment (YoshithaRathnayake) [reply dark/index.html:196]: No no you have misunderstand it, I asked why did you keep? I didn't ask why did you remove them??? You have to remove all the formContainers

Comment (PiyushXCoder) [reply dark/index.html:196]: So, Should I remove Those again?

File: dark/index.html [hunk 3]
```diff
@@ -194,21 +194,6 @@ <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3>
 
       <div class="formContainer bg-dark p-5 rounded-3">
```
Comment (YoshithaRathnayake) [inline dark/index.html:196]: Remove this line also and the related css codes @PiyushXCoder

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 2 above)
Comment (PiyushXCoder) [reply dark/assets/css/styles.css:1049]: I have pushed the changes. This is what it looks like.

Screenshot 2024-10-08 at 16-58-41 YOSHITHA RATHNAYAKE

File: dark/index.html (hunk 3 above)
Comment (PiyushXCoder) [reply dark/index.html:193]: I have pushed the changes. This is what it looks like.

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Don't remove the title and stuff just remove the form??? @PiyushXCoder

**Comment by YoshithaRathnayake** [comment]
> Don't remove the title and stuff just remove the form??? @PiyushXCoder

Close this PR and make a new one (remove the codes only related to the contact form) @PiyushXCoder

**Comment by PiyushXCoder** [comment]
I am not continuing with the contribution. I lost my Interest. You do keep changing the requirements.

**Comment by YoshithaRathnayake** [comment]
> I am not continuing with the contribution. I lost my Interest.

Can you understand english? @PiyushXCoder 

image

I told to remove this, You have removed all 😡

**Comment by PiyushXCoder** [comment]
I did remove it first then you asked me to get it back [Here](https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/pull/228#discussion_r1791070375)
Later I add it which you said me to remove it again [Here](https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/pull/228#discussion_r1791516953)
Please be clear about your requirements.

**Comment by YoshithaRathnayake** [comment]
image

I asked why did you keep these lines??? You have removed the topic also, is that the contact form ah???

**Comment by YoshithaRathnayake** [comment]
image

I told to remove these lines but you have removed the topic??? What are you telling???

**Pull request closed without merging**